integration-cli/docker_cli_build_test.go

```diff
@@ -5920,6 +5951,51 @@ func (s *DockerSuite) TestBuildCopyFromPreviousFrom(c *check.C) {
 	c.Assert(strings.TrimSpace(out), check.Equals, "def")
 }
 
+func (s *DockerSuite) TestBuildCopyFromImplicitFrom(c *check.C) {
+	dockerfile := `
+		FROM busybox
+		COPY --from=busybox /etc/passwd /mypasswd
+		RUN cmp /etc/passwd /mypasswd`
```

Comment:
@tonistiigi seeing as `COPY --from` is not case-sensitive, is it worth covering that in a test to further document/prove? I read the PR but didn't see one..